HHS/Head-Start-TTADP
# [TTAHUB-2813] Daily DB backup by CI

## Description of change
Uses CI to back up prod db to s3 in a password protected zip file


## How to test
view successful execution in ci & verify the files are in s3 under ttahub-db-backups instance

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-2813


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
